index.js

```diff
@@ -145,7 +145,7 @@ const runChecks = async (context, checkEvent) => {
             callable.push(...[
               periodicCheckModule.ensureAllPullRequestsAreAssigned(context),
               periodicCheckModule.ensureAllIssuesHaveProjects(context),
-              staleBuildModule.checkAndTagPRsWithOldBuilds(context),
+              staleBuildModule.checkAndTagPRsWithOldBuilds(context)

```

Comment:
Unneeded change.

Reply:
Corrected it in the Latest Commit Sir.Thanks.